### ISSUE TITLE
How to ensure the consistency of the index， during training stage and prediction stage?

### ISSUE THREAD

**Issue by paulxu1314** [shenweichen/DeepCTR-Torch#306]
lbe = LabelEncoder()
data[feat] = lbe.fit_transform(data[feat])
example: 
in traing stage, feat_1:['a', 'b', 'c']
in prediction stage, feat_1:['a', 'c', 'd']

**Comment by zanshuxun** [comment]
As shown in the example, a label encoder is used to re-encode the features in the entire dataset `data`, and then `data` is divided into a `train` and `test`, so that the mapping relationship of the feature values ​​is consistent in the training set and the test set.
https://github.com/shenweichen/DeepCTR-Torch/blob/f6854257b1fc29caab655ce72c6b5528b7432574/examples/run_classification_criteo.py#L24
